docs/_static/code/getting_started/moviepy_10_minutes/trailer.py

```diff
@@ -0,0 +1,307 @@
+# Lets import moviepy, lets also import numpy we will use it a some point
+from moviepy import *
+import numpy as np
+
+
+#################
+# VIDEO LOADING #
+#################
+# We load our video
+video = VideoFileClip("./resources/bbb.mp4")
+
+
+#####################
+# SCENES EXTRACTION #
+#####################
+# We extract the scenes we want to use
+
+# First the characters
+intro_clip = video.with_subclip(1, 11)
+bird_clip = video.with_subclip(16, 20)
+bunny_clip = video.with_subclip(37, 55)
+rodents_clip = video.with_subclip(
+    "00:03:34.75", "00:03:56"
```

Comment:
why not recommend the pythonic slicing approach from #1076 ? 

```suggestion
intro_clip = video[1, 11]
bird_clip = video[16, 20]
bunny_clip = video[37, 55]
rodents_clip = video["00:03:34.75", "00:03:56"]
```

Reply:
Hi, two main reasons :
- I personally dont like it
- This is not a standard python code and more of a specific syntax allowed only by the project handling of python operators, meaning it can be confusing for new comers and coders coming from another language.

Don't take me wrong I see the appeal of the pythonic way in a daily usage (though I dont necessarily agrees with it), but for the documentation I think its best to keep things as simple and generalist as possible.